source/administration/image-proxy.rst

```diff
@@ -6,10 +6,6 @@ servers, tracking pixels is allowed. Users can put invisible images in posts tha
 for every user that views the post. An image proxy protects user privacy by eliminating this direct interaction with 
 third-party servers.
 
-Many proxy servers also provide resizing-as-a-service. For example, if you use `atmos/camo image proxy server <https://github.com/atmos/camo>`_, you can configure 
-``ImageProxyOptions`` to resize images if they're more than ``x`` pixels tall. Users can save bandwidth by downloading 
-only the resolution they need.
-
 Proxy servers also provide a layer of caching, and can be made faster and more reliable than third-party sites. This caching 
 also helps preserve posts by protecting them from dead images.
 
```

Comment:
Below: 

> Additional options for basic image adjustments such as resizing, cropping and rotation.

The description in-app was changed to say "Additional options such as the URL signing key."

> The image proxy prevents users from making insecure image requests to Mattermost, provides caching for increased performance, and automates image adjustments such as resizing.

The previously insecure image requests weren't made to Mattermost. They were made to third-party sites. And we should remove mention of resizing from here. So something like this might be better:

> The image proxy prevents users from making insecure image requests to third-party sites and provides caching for increased performance.

Reply:
Thanks, fixed.